### ISSUE TITLE
A breakdown of how a specific lib is used across specific projects

### ISSUE THREAD

**Issue by mascardoso** [infusionsoft/sleuth-sketch#6]
Hi.
Not sure if this is something that we can already do or not.
But as the title suggests I'm more interested in understanding the breakdown of how a **specific lib** (our design system) is used across **specific** projects.

It's a great tool (congrats on that), but for our use case, not understanding what are the external libs used on those files it just doesn't make sense to use.

**Comment by faridsabitov** [comment]
Yes, that would be really helpful! Is there a chance to do that? Thank you for such an amazing tool!

**Comment by seanriceaz** [comment]
Hi there @mascadoso and @faridsabitov ! I've done some work to start logging which components are being used, and in which files. I don't have a frontend for the data yet (it could be some time before I do), but if you get the latest version of Sleuth and run the report, you can get that information in the JSON files now.

I know that's not a full update for you, but I'm on the way. Happy to merge a PR from you if you can resolve the frontend!

**Comment by mascardoso** [comment]
@seanriceaz I will gladly help you on the GUI side of things. Next week I'm planning to do so.
Thank you

**Comment by andrew-callaghan** [comment]
@seanriceaz thanks again for helping us get started with Sleuth! I think we have similar challenge in identifying which components in which files are being used. You mention that the information is in the JSON files now, can anyone please confirm where the JSON files are stored? Thanks!

**Comment by nunogois90** [comment]
Hi @seanriceaz, first of all, we're quite excited with the possibilities Sleuth opens for us as a Design System team. It's amazing, thanks for making it happen!
We're quite curious if you have any update on a potential breakdown by the component that you mentioned in the above message.
